pkg/controller/cassandradatacenter/cassandradatacenter_controller.go

```diff
@@ -122,8 +122,13 @@ func (r *ReconcileCassandraDataCenter) Reconcile(request reconcile.Request) (rec
 	}
 
 	statefulSet, err := createOrUpdateStatefulSet(rctx, configVolume)
-	if err != nil && err != ErrorCDCNotReady {
-		return reconcile.Result{}, err
+	if err != nil {
```

Comment:
What happens, in theory, when other event is triggered (for whatever reason) before one minute passes?

Reply:
Nothing special, it will be put into the "queue" to be worked on. Basically, all events that trigger the handler are the same from our perspective, we don't know what initiated them, but they all become the same "request" with name and namespace.

So in this case, if there are 2 or more "competing" calls in the queue, the first one to get to a state where the cluster is "ready" will win and cause some action. Then the other one will be still in the queue, waiting until worked on, meaning that either there's some reconcile will be happening or nothing will happen (meaning nothing left to reconcile). Then the call will be cleared from the queue.